<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Q_Wordreciting</class>
 <widget class="QWidget" name="Q_Wordreciting">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>600</width>
    <height>400</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>千词斩 Version 1.0</string>
  </property>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>150</x>
     <y>30</y>
     <width>300</width>
     <height>80</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>微软雅黑</family>
     <pointsize>20</pointsize>
    </font>
   </property>
   <property name="text">
    <string>峰峰的日常</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QPushButton" name="button_Back">
   <property name="geometry">
    <rect>
     <x>510</x>
     <y>350</y>
     <width>75</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>返回</string>
   </property>
  </widget>
  <widget class="QPushButton" name="button_CET6">
   <property name="geometry">
    <rect>
     <x>160</x>
     <y>200</y>
     <width>120</width>
     <height>50</height>
    </rect>
   </property>
   <property name="text">
    <string>CET-6</string>
   </property>
  </widget>
  <widget class="QPushButton" name="button_CET4">
   <property name="geometry">
    <rect>
     <x>160</x>
     <y>120</y>
     <width>120</width>
     <height>50</height>
    </rect>
   </property>
   <property name="text">
    <string>CET-4</string>
   </property>
  </widget>
  <widget class="QPushButton" name="button_TOEFL">
   <property name="geometry">
    <rect>
     <x>160</x>
     <y>280</y>
     <width>120</width>
     <height>50</height>
    </rect>
   </property>
   <property name="text">
    <string>TOEFL</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_CET4_Count">
   <property name="geometry">
    <rect>
     <x>330</x>
     <y>120</y>
     <width>200</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>总单词数量：100 </string>
   </property>
  </widget>
  <widget class="QLabel" name="label_CET4_Progress">
   <property name="geometry">
    <rect>
     <x>330</x>
     <y>150</y>
     <width>200</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>进度：10%</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_CET6_Count">
   <property name="geometry">
    <rect>
     <x>330</x>
     <y>200</y>
     <width>200</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>总单词数量：100</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_CET6_Progress">
   <property name="geometry">
    <rect>
     <x>330</x>
     <y>230</y>
     <width>200</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>进度：10%</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_TOEFL_Count">
   <property name="geometry">
    <rect>
     <x>330</x>
     <y>280</y>
     <width>200</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>总单词数量：100</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_TOEFL_Progress">
   <property name="geometry">
    <rect>
     <x>330</x>
     <y>310</y>
     <width>200</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>进度：10%</string>
   </property>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
